AWSome.Bucket/Config/BucketException.cs

```diff
@@ -1,10 +1,18 @@
 ﻿using Amazon.S3;
 using System;
 
-namespace AWSome.Bucket.Config
+namespace Liberator.AWSome.Bucket.Config
 {
+    /// <summary>
+    /// 
+    /// </summary>
     public class BucketException : Exception
     {
+        /// <summary>
+        /// 
```

Comment:
summay is empty?

Reply:
would help if I spelt it correctly - *summary

Reply:
This bucket is definitely exceptional